cypress/fixtures/create_post_no_success.json

```diff
@@ -0,0 +1,8 @@
+{
+  "status": 422,
+  "headers": {},
+  "error": [
+    "Caption is too long (maximum is 140 characters)",
+    "You need to upload an image"
+  ]
+}
```

Comment:
Do you really need `headers` and `status` in the fixture?

Reply:
Tested without and the test passes anyway. Since we are not testing for them in the response in the integration file, we don't need them